worlds/animal_well/region_data.py

```diff
@@ -204,14 +203,14 @@ class AWData(NamedTuple):
     },
     rname.fish_west: {
         rname.fish_wand_pit: 
-            AWData(AWType.region, [[iname.bubble], [iname.disc], [iname.wheel, iname.weird_tricks]]),
+            AWData(AWType.region, [[iname.bubble], [iname.disc], [iname.wheel]]),
```

Comment:
No obscure tricks here?

Reply:
We were flip-flopping on it. I feel like "use wheel to artificially expand your ground grab distance" isn't really a trick. You can just roll of the ledge with wheel and it works. I'm not OPPOSED to it being obscure by any means, if you really prefer it.

Reply:
I'm asking a friend who isn't in the animal well channel to try and figure out what makes sense here.
Also, water bounce should probably be here too

Reply:
Obscure tricks it is -- they got it easy with bubble and disc, and tried and failed with wheel and assumed that was the trick logic I was mentioning to them

Reply:
I'll get it committed.